### PR TITLE
[DOC] Tutorial improvements

### LINKED ISSUE #2253: Tutorial: wrong parameter in example code
## Platform

  - SeqAn tutorial: http://docs.seqan.de/seqan/3-master-user/usergroup1.html

### Which section of the tutorial

https://github.com/seqan/seqan3/blob/master/doc/tutorial/ranges/views/minimiser/index.md

### Expected example

An example where k-mer size and window size are equal and we are not considering the backward strand.

`auto minimisers = text | seqan3::views::kmer_hash(seqan3::ungapped{4}) | seqan3::views::minimiser(1);`

### Actual example

K-mer size is 4 and window size is 8.

`auto minimisers = text | seqan3::views::kmer_hash(seqan3::ungapped{4}) | seqan3::views::minimiser(5);`

### PR DESCRIPTION
**Pull request by eaasna**
Fixes #2253 and some minor phrasing errors

### CODE REVIEW

**Review by smehringer — COMMENTED** [review]
One small thing I forgot to tell you, sorry!! We prefix our commits with tags, e.g. `[DOC] blah blah.`
Here is a list of the tags: 
`[DOC] ... - all stuff only related to the documentation or tutorials`
`[TEST] ... - all stuff only related to tests or benchmarks`
`[FIX] ... - a bug fix (concerning code, not typos or documentation)`
`[INFRA] ... - all stuff only related to strucural changes (e.g. CMake and CI)`
`[FEATURE] ... - when something new was added that is relevant to users`
`[MISC] ... - everything else that does not fit the above`

File: doc/tutorial/ranges/views/minimiser/minimiser_snippets.cpp [hunk 1]
```diff
@@ -17,6 +17,7 @@ auto minimisers = text | seqan3::views::kmer_hash(seqan3::ungapped{4}) | seqan3:
 //![minimiser]
 }
 
+
```
Comment (joergi-w) [inline doc/tutorial/ranges/views/minimiser/minimiser_snippets.cpp:20]: Please do not insert more than one empty line.

File: doc/tutorial/ranges/views/minimiser/index.md [hunk 2]
```diff
@@ -112,17 +112,18 @@ the backward strand should not be considered. If this is the desired behaviour t
...
+
 
 This syntax will result in minimisers with k-mer size 4 and a window-length of 8 (5 + 4 - 1). (So, to determine the
 right window size you always have to determine the window size you would like to use in `seqan3::views::minimiser_hash`
 and subtract it by the k-mer size and add 1, here: 8 - 4 + 1 = 5.)
 Got it? Then let's make a simple test, what is the actual window size of the following:
 
-\snippet doc/tutorial/ranges/views/minimiser/minimiser_snippets.cpp minimiser
 
+```cpp
```
Comment (joergi-w) [inline doc/tutorial/ranges/views/minimiser/index.md:124]: There are two consecutive empty lines above, please remove one of them.

**Review by joergi-w — CHANGES_REQUESTED** [review]
Thank you for correcting our tutorial! 
I just found two code style issues and I agree that the commit messages should have a [DOC] tag as prefix.

File: doc/tutorial/ranges/views/minimiser/index.md (hunk 2 above)
Comment (eaasna) [reply doc/tutorial/ranges/views/minimiser/index.md:124]: Thanks, didn't notice that :) please review again

**Review by joergi-w — APPROVED** [review]
Thank you for deleting the newline!

Please rename the commit messages with [DOC] prefix for **each** of your commits in this Pull Request. This is easy if you `git rebase -i master` and then use _reword_ (instead of _pick_) for the respective commits.

**Comment by eaasna** [comment]
> Thank you for deleting the newline!
> 
> Please rename the commit messages with [DOC] prefix for **each** of your commits in this Pull Request. This is easy if you `git rebase -i master` and then use _reword_ (instead of _pick_) for the respective commits.

Thanks for the explanation. Commit messages have now also been fixed

**Review by joergi-w — APPROVED** [review]

**Review by smehringer — APPROVED** [review]

**Pull request merged**